modules/apps/notification/notification-test/src/testIntegration/java/com/liferay/notification/service/test/NotificationTemplateServiceTest.java

```diff
@@ -311,6 +261,10 @@ private void _testUpdateNotificationTemplate(User ownerUser, User user)
 	@Inject
 	private CounterLocalService _counterLocalService;
```

Comment:
Since you refactored this class, I believe these variables are not being used anymore: _counterLocalService, _notificationQueueEntryLocalService, _notificationRecipientLocalService, _notificationRecipientSettingLocalService;

Could you please double-check? If positive, could you please remove it?